### ISSUE TITLE
Workspaces and disabling / enabling monitors.

### ISSUE THREAD

**Issue by pranjlp** [vaxerski/Hyprland#149]
**Describe the bug**
On disabling a monitor.. The workspace assigned remains assigned to the disabled monitor. Is it possible allow a config or maybe consider changing from this as the default behavior. The alternative being to move the affected workspace to one of  the enabled monitors.

On enabling a monitor, the first unoccupied workspace( or the 10th workspace, if all of the workspace have windows) seems to be assigned automatically to it. This behavior is acceptable. Please document how this works. Its good to know how this works in back of your head if one works with projectors and presentations

**Images/videos/anything that would help**
```
hyprctl workspaces
workspace ID 1 (1) on monitor eDP-1:
	windows: 3
	hasfullscreen: 0

workspace ID 9 (9) on monitor eDP-1:
	windows: 2
	hasfullscreen: 0

workspace ID 10 (10) on monitor eDP-1:
	windows: 1
	hasfullscreen: 0

workspace ID 4 (4) on monitor HDMI-A-1:
	windows: 0
	hasfullscreen: 0
```
The eDP monitor is disabled and yet workspaces remained assigned to them..

Please attach a log (it's in /tmp/hypr/hyprland.log) and, if crashing, a crashdump (coredumpctl, then coredumpctl info <pid>)
[log](https://pastebin.com/rZfVmfpE) if needed

*Please do NOT attach coredumps from any packaged version of Hyprland (AUR, etc.). Compile manually, and THEN attach a coredump of that.*

**Comment by vaxerski** [comment]
please remember disabling a monitor is permanent until a restart.

Should be fixed in 4f76d5d8d73e29e0b291db68916f4c846bd50706

**Comment by vaxerski** [comment]
closing due to inactivity
